src/main/java/org/opensearch/ad/task/ADTaskManager.java

```diff
@@ -196,6 +197,8 @@
 public class ADTaskManager {
     public static final String AD_TASK_LEAD_NODE_MODEL_ID = "ad_task_lead_node_model_id";
     public static final String AD_TASK_MAINTAINENCE_NODE_MODEL_ID = "ad_task_maintainence_node_model_id";
+    // HC batch task timeout after 10 minutes
```

Comment:
Is this per task or per detector (i.e., total processing time of the 1K entity tasks should be less than 10 mins)?  According to ADTaskManager.resetHistoricalDetectorTaskState, this sounds like the latter. If so, can we assume all historical analyses finishes within 10 minutes?